packages/web/src/pages/upload-page/forms/EditCollectionForm.tsx

```diff
@@ -113,7 +119,9 @@ export const EditCollectionForm = (props: EditCollectionFormProps) => {
             </div>
           </div>
           <ReleaseDateFieldLegacy />
-
+          {isAlbum && isPremiumAlbumsEnabled ? (
```

Comment:
I'm now forgetting the exact feature flag plan.

We want the ability to upload premium albums to be whitelist only, but once uploaded, all users should be able to access and purchase premium content (post-launch).

We can either:
Gate visibility of premium albums on one flag and have a separate flag for upload access (only granted to the whitelisted audiuence)
Or don't gate visibility of the content, but use the feature flag to gate upload (only granted to whitelist)

Reply:
We have 2 feature flags:
* `PREMIUM_ALBUMS_ENABLED`: should use this to filter out premium albums from lineups
* `USDC_PURCHASES_UPLOAD`: should use this for upload access - this is the one that is linked to the `[Phase 1 Monetization - Early Access](https://app.optimizely.com/v2/projects/18140642080/audiences/25558290311/)` audience on optimizely